make/command

```diff
@@ -2,13 +2,13 @@ ifeq ($(CMDSTAN_SUBMODULES),1)
 bin/cmdstan/stansummary.o : src/cmdstan/stansummary_helper.hpp
 bin/cmdstan/%.o : src/cmdstan/%.cpp
 	@mkdir -p $(dir $@)
-	$(COMPILE.cpp) -fvisibility=hidden $(OUTPUT_OPTION) $(LDLIBS) $<
+	$(COMPILE.cpp) -fvisibility=hidden $< $(OUTPUT_OPTION)
```

Comment:
Was this change intended? I think it's a good change: the use of libs shouldn't be necessary when compiling, only linking.

Reply:
Yes, it currently generates warnings for unused arguments, so I removed it